apm-agent-plugins/apm-log-shipper-plugin/src/test/java/co/elastic/apm/agent/log/shipper/ApmServerLogShipperTest.java

```diff
@@ -40,6 +40,8 @@
 import org.junit.Before;
 import org.junit.Rule;
 import org.junit.Test;
+import org.junit.jupiter.api.condition.DisabledOnOs;
+import org.junit.jupiter.api.condition.OS;
```

Comment:
Those were not applied because they are JUnit 5, whereas the `@Test` and lifecycle (e.g. `@Before`) annotations are JUnit 4. 
Upgrading to JUnit 5 will require a [migration](https://junit.org/junit5/docs/current/user-guide/#migrating-from-junit4-rule-support) from `@Rule` though. If you can turn this into a JUnit-5-"native" test class, that would be best. Otherwise, using the migration annotation or leaving as is and leave it for us is OK as well.

Reply:
Fixed